plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -1022,9 +1022,6 @@ public static OrcWriterOptions getOrcWriterOptions(Properties schema, OrcWriterO
             }
             try {
                 double fpp = parseDouble(schema.getProperty(ORC_BLOOM_FILTER_FPP));
-                if (fpp > 0.0 && fpp < 1.0) {
```

Comment:
Is this check invalid or redundant?  If redundant, then state that instead of saying invalid.  If invalid, state why instead of saying there is a similar check somewhere.... either way update the commit message.

Reply:
applied comments.